helix-core/src/test/java/org/apache/helix/tools/TestClusterSetup.java

```diff
@@ -356,51 +360,65 @@ public void testDropInstance() throws Exception {
         "MasterSlave", true); // do rebalance
 
     // add fake liveInstance
-    ZKHelixDataAccessor accessor =
-        new ZKHelixDataAccessor(clusterName, new ZkBaseDataAccessor<ZNRecord>(ZK_ADDR));
```

Comment:
Does this mean if user constructs a ZKHelixDataAccessor with a ZK-ADDR, they can create an ephemeral node before, but not anymore after they bump to 1.0.0?

Reply:
@lei-xia No, it doesn't. Before 1.0.0,  a ZKHelixDataAccessor with a ZK-ADDR is not able create an ephemeral node. It is also by default using a shared zkclient internally. The only difference is the type of exceptions: before 1.0.0, `HelixException` is thrown from sharedZkClient, while in 1.0.0, `UnsupportedOperationException` is thrown.

The root cause for this test is, even though a live instance(ephemeral node) is not created by the ZKHelixDataAccessor with a ZK-ADDR, the test still passes, because the test doesn't catch or verify the correct result.